storage/gcloud/aio/storage/storage.py

```diff
@@ -172,6 +173,37 @@ async def _headers(self) -> Dict[str, str]:
             'Authorization': f'Bearer {token}',
         }
 
+    # This method makes the following API call:
+    # https://cloud.google.com/storage/docs/json_api/v1/buckets/list
+    async def list_buckets(
+        self, project: str, *,
+        params: Optional[Dict[str, str]] = None,
+        headers: Optional[Dict[str, Any]] = None,
+        session: Optional[Session] = None,
+        timeout: int = DEFAULT_TIMEOUT,
+    ) -> Set[Bucket]:
+        url = f'{self._api_root_read}?project={project}'
+        headers = headers or {}
+        headers.update(await self._headers())
+        if not params:
+            params = {'pageToken': ''}
```

Comment:
I'm not 100% sure on the semantics of this API from reading the docs, but given you're setting the pageToken here explicitly, it seems like perhaps this value needs to be provided?

As such, there might be a bug here where if the user provides `params` but the `params` _don't_ include a pageToken, we end up in a broken state?

As such, we might want to prefer doing something like this:
```suggestion
        params = params or {}
        if not params.get('pageToken'):
            params['pageToken'] = ''
```

Reply:
I had to initialize the `params` dict  and set `params[pageToken]` to be `''` because I couldn't  ask for:
```
if not params['pageToken']
```
I'll use your three lines to have a more elegant solution